extensions/convert-px-to-vw-vh/src/menu-bar.tsx

```diff
@@ -0,0 +1,44 @@
+import { Clipboard, MenuBarExtra, openCommandPreferences, showHUD } from "@raycast/api";
+import { useHistory } from "./history";
+
+export default function Command() {
+  const { history, remove, clear } = useHistory();
+  const icon = "icon.png";
```

Comment:
there's no `icon.png` in the `assets/` folder, so this doesn't show up in the menu bar:
<img width="254" alt="image" src="https://github.com/raycast/extensions/assets/1155589/36f0e7fa-1183-4025-bb84-f5e2e97f31e4">